src/crab/type_to_num.cpp

```
@@ -198,14 +207,12 @@ TypeToNumDomain::join_over_types(const Reg& reg,
 }
 
 void TypeToNumDomain::havoc_all_locations_having_type(const TypeEncoding type) {
```

Cppcheck (2.20.0)

[style] The function 'havoc_all_locations_having_type' is never used.

(unusedFunction)